api/v1beta2/foundationdbcluster_types.go

```diff
@@ -298,8 +298,10 @@ type MaintenanceModeInfo struct {
 	// StartTimestamp provides the timestamp when this zone is put into maintenance mode
 	StartTimestamp *metav1.Time `json:"startTimestamp,omitempty"`
 	// ZoneID that is placed in maintenance mode
+	// +kubebuilder:validation:MaxLength=512
 	ZoneID string `json:"zoneID,omitempty"`
 	// ProcessGroups that are placed in maintenance mode
+	// +kubebuilder:validation:MaxItems=10000
```

Comment:
```suggestion
	// +kubebuilder:validation:MaxItems=200
```

I would prefer to have a more reasonable value here. I would like to see the FDB cluster that has 10000 processes per zone :)